examples/panorama/example.tfvars

```diff
@@ -1,11 +1,11 @@
 # General
-project_id = "gcp-gcs-pso"
-project    = "gcp-gcs-pso"
+project_id = "example-panorama"
+project    = "example-panorama"
 region     = "us-central1"
 
 # VPC
-vpc_name          = "panorama-example-vpc"
-subnet_name       = "panorama-example-subnet"
+vpc_name          = "example-panorama-vpc"
+subnet_name       = "example-panorama-subnet"
 cidr              = "172.21.21.0/24"
 allowed_sources   = ["0.0.0.0/0"]
```

Comment:
Maybe we should create variable from that, and don't default it to 0.0.0.0/0 :D

Reply:
Actually, a variable is already there, I will update the value to sth that always has to be replaced.